### PR TITLE
Add delete features.

### PR DESCRIPTION
**Pull request by saikatgpae**
This PR is created to add delete all completed tasks.

### CODE REVIEW

File: src/common/lists.js [hunk 1]
```diff
@@ -14,8 +14,11 @@ export class List {
 
   add(description, completed) {
     if (Array.isArray(this.list)) {
-      const index = this.list.length;
+      const storedTodoList = JSON.parse(localStorage.getItem('toDolist')) ?? [];
```
Comment (elmar8287) [inline src/common/lists.js:17]: - [x] In this file you have linter disabled - please, consider removing it. Let the linter makes its direct job - find the bugs in the early stages of development 👍

File: src/index.js [hunk 2]
```diff
@@ -21,12 +21,12 @@ document.querySelector('.refresh').addEventListener('click', refresh);
 // ADD NEW TO-DO LIST
```
Comment (elmar8287) [inline src/index.js:22]: - [x] Please, consider removing comments from your code, to make it clear. Your code is understandable enough and does not need any comments - it is talking itself about what happening 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @saikatgpae.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Clear code 👍 
- Nice README file.

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/common/lists.js (hunk 1 above)
Comment (saikatgpae) [reply src/common/lists.js:17]: Corrected the same. As there is 2 class is declared, so 1 class is disabled.

File: src/index.js (hunk 2 above)
Comment (saikatgpae) [reply src/index.js:22]: Removed the same.

**Review by Lordkaito — APPROVED** [review]
Hi @saikatgpae, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**